plugins/logger/api/api.js

```diff
@@ -2,9 +2,22 @@ var exported = {},
     common = require('../../../api/utils/common.js'),
     plugins = require('../../pluginManager.js');
 
+plugins.setConfigs("logger", {
+    state: "on"
```

Comment:
Any reason why you went with the string "on/off" instead of boolean true/false?
I think boolean would also provide input out of the box too, so you would not have to implement your own.

Reply:
As we discussed. Since we are thinking of introducing a new state that will automatically disable the logging og requests when a certain amount is reached, I kept it as string values.